presto-spi/src/main/java/io/prestosql/spi/security/ConnectorIdentity.java

```diff
@@ -14,7 +14,6 @@
 package io.prestosql.spi.security;
 
 import java.security.Principal;
-import java.util.Objects;
```

Comment:
please squash this with the previous commit that removed the usage of this import